### ISSUE TITLE
Game freeze upon reaching max burnout (high magic level only)

### ISSUE THREAD

**Issue by Lykayos** [TCLProject/ArsMagica2-5#52]
Issue takes place when AM2.5 is inserted into a modpack (The 1.7.10 Pack) Unsure if issue persists with no other mods around.

If in a server environment the game will freeze and must be reloaded for the issue to resolve.

If in a single player environment the player will become a ticking entity and the single player world may never be entered again.

Problem only occurs when max burnout is reached, requesting a Config to revert burnout changes to original AM2 for modpack integration of AM2.5

No crash report generated unless trying to join a single player world affected by the event.

![image](https://github.com/user-attachments/assets/7f6b768a-c82a-4a1c-81af-38e0e80876f6)
^ This is the permanent result of the player when burnout is reached.

[crash-2024-09-11_11.20.34-client.txt](https://github.com/user-attachments/files/16968897/crash-2024-09-11_11.20.34-client.txt)
^ Attempting to rejoin a world affected by burnout freeze will result in this crash report.

At lower magic levels, the player takes damage and gets affected by ALL of the burnout issues all at once, immunity is never reached no matter what level you are, the effects just get worse.

**Comment by Lykayos** [comment]
Upon reaching Magic Level 70, reaching max burnout just instantly kills you on occasion.
Getting higher levels causes the above half dead state.

**Comment by AirBurn0** [comment]
I myself noticed this problem about a year ago, so I know how to reproduce it.
Looks like problem already has been fixed by https://github.com/DrParadox7/ArsMagica2/pull/4, and there is no reason for me to interfere.

**Comment by Lykayos** [comment]
Yeah found out last night by asking around the TCL discord, but air burn
may I ask a favor of you?

Would you be willing to teach me how to compile it into a workable jar file?

**Comment by AirBurn0** [comment]
> Yeah found out last night by asking around the TCL discord, but air burn may I ask a favor of you? Would you be willing to teach me how to compile it into a workable jar file?

Oh, I see. They dont actually BUILD these changes.
Usually a working build (compiled .jar) is provided by the one who made the changes (@RubilaxXxx), or by the repo maintainer (@DrParadox7).
There is no need for you to learn how to compile java code, since this is not your job, and in a couple of days @DrParadox7 may post a working build himself...
Alternatively, just ask @RubilaxXxx to do so.
Of course, I can build a project myself, but I don't want to do it right now.

**Comment by Lykayos** [comment]
Sounds good, thanks for your time AirBurn

**Comment by RubilaxXxx** [comment]
[fixed here btw](https://github.com/RubilaxXxx/ArsMagica2-5/releases/tag/v.1.7.0-RE)
